readthedocs/audit/filters.py

```diff
@@ -0,0 +1,41 @@
+"""Filters used in our views."""
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from django_filters import CharFilter, ChoiceFilter, FilterSet
+
+from readthedocs.audit.models import AuditLog
+
+
+class _UserSecurityLogForm(forms.ModelForm):
+
+    """
+    Form to be used in the filters.
+
+    As we use an ``IPAddressField`` in our model,
```

Comment:
We could also change our model to be a charfield instead...

Reply:
I think using a Charfield seems reasonable, unless the IPAddressField gives us something valuable we don't have?

Reply:
From what I found, it only gives us validation and normalization of values. Since we get the IP field from internal headers, I think we should be fine with a charfield, don't think a user could pass an invalid IP down there :thinking:

https://docs.djangoproject.com/en/3.2/ref/models/fields/#genericipaddressfield